### DISCUSSION TITLE
[Bug]: Map row displayed when the map is not displayed

### DISCUSSION THREAD

**Discussion by Oclair** [code4recovery/12-step-meeting-list#1275] [Q&A] [answered]
### Contact Details

feedback@aavienna.com

### Website With Issue

https://www.aavienna.com

### What happened?

Hi,

On published meeting pages; The table row to display a map is also not restricted when the map display is restricted.  
On location pages; The table row to display a map is also not restricted when the map display is restricted.  

This might be resolved by changing the behavior to either display the full earth map, the city and country provided which is required to publish the meeting, or simply remove the row entirely.  

As it is, the meeting and location page rendering is broken as the is a small row packed with meeting details and a large row empty where the map would normally display.  The table's application is broken.

Kind Regards and thank you for your service,
Jason D

**Comment by joshreisner** [comment]
Hi Jason,

When you say "When the map display is restricted" - what does that mean?

I'm trying to figure out why we're not seeing maps on your meeting and location detail pages, for example https://www.aavienna.com/locations/evangelische-auferstehungskirche-church/ but we do see it on the main page, eg https://www.aavienna.com/meetings/?tsml-view=map

Looks like you have a mapbox key set, and these locations do have geographic coordinates. 

Thanks

**Reply by joshreisner** [reply]
Looks like you are using a plugin called Complianz GDPR to restrict 12 Step Meeting List from showing the map - this plugin thinks you are using Google Maps when you are in fact using Mapbox. Please uncheck the Google Maps restriction in Complianz and the map should start working again. Thanks

**Reply by Oclair** [reply]
Thank you Josh, I will look into this and report back.  Although what you wrote, it indeed seems very likely.  Hope your having a nice weekend!  Jason

**Reply by Oclair** [reply]
@joshreisner  I disabled Complianz and the behaviour remains where the mapbox does not display for online meeting pages, while the face to face meetings have no issue displaying a map.

![image](https://github.com/code4recovery/12-step-meeting-list/assets/619643/10dd9775-afa7-4abd-857d-249e439e45ad)

vs

![image](https://github.com/code4recovery/12-step-meeting-list/assets/619643/2d3cb599-6835-4b25-8c44-f9e02f454c4c)

**Reply by joshreisner** [reply, marked as answer]
great, im glad the in person meetings are fixed

as for the blank space on online meetings, that's how it looks. if you switch to the TSML UI appearance we have a placeholder image there. example: https://aasanjose.org/meetings?meeting=f-f-a-living-sober-unity-group

**Reply by Oclair** [reply]
I am getting `Timezone +02:00 is not valid. Please use one like Europe/Rome` after switching to it

![image](https://github.com/code4recovery/12-step-meeting-list/assets/619643/470b433a-c44f-4833-a87c-c5720a56dfd9)

**Reply by Oclair** [reply]
I am currently trying to figure out if there is an error message some wheres for you

**Reply by Oclair** [reply]
For future reference for anyone who might get this error Wordpress in wp-admin/options-general.php also provides specific cities and regions instead of the UTC +- which the plugin requires for the new UI!

Great work on the UI @joshreisner
